packages/manager/modules/pci/src/projects/project/sidebar/sidebar.scss

```diff
@@ -0,0 +1,4 @@
+.oui-status.oui-badge_new {
```

Comment:
Not a blocker but I think it might be better to move this badge
directly here: https://github.com/ovh/ovh-ui-kit/blob/v2.x/packages/ovhcloud-light-theme/components/_sidebar.less#L146-L164

It will allow us to not having to deal with two modifier that can collid.

cc @AxelPeter